src/Compilers/CSharp/Test/Semantic/IOperation/IOperationTests_IBinaryOperatorExpression.cs

```diff
@@ -34,6 +34,112 @@ void F(int? x, int? y)
             VerifyOperationTreeForTest<BinaryExpressionSyntax>(source, expectedOperationTree);
         }
 
+        [CompilerTrait(CompilerFeature.IOperation)]
+        [Fact]
+        public void VerifyLiftedBinaryOperators2()
```

Comment:
Not seeing equivalent added tests for VB.

Reply:
> Not seeing equivalent added tests for VB.

You are correct. I didn't need to add extra tests for this in VB because I didn't have to synthesize nodes for similar VB scenarios in ControlFlowGraphBuilder.

---
In reply to: [184243761](https://github.com/dotnet/roslyn/pull/26395#discussion_r184243761) [](ancestors = 184243761)